content/docs/reference/config.mdx

```diff
@@ -56,3 +56,37 @@ export default defineConfig({
 
 In this example, the TinaCMS admin will be viewable at `<my-site-url>/admin/index.html`.
 For more information [check out the content modeling section](/docs/schema/).
+
+
+## Typescript Path Alias 
+
+TinaCMS supports TypeScript path aliases in the `tina.config.js` file and collections, making it easier to organize and maintain your code.
```

Comment:
**question**: Should the file specified be the `tina/config.js` or `tina/config.ts` file?

The latter would be consistent with the caption below.

```suggestion
TinaCMS supports TypeScript path aliases in the `tina/config.ts` file and collections, making it easier to organize and maintain your code.
```

Reply:
I guess the answer for that is "tina/config.{ts,js,tsx}"

Reply:
Done - reword all in to tina/config.{ts,js,tsx}